### PR TITLE
SSC::Process: fix high cpu in wait()

### PR DESCRIPTION
**Pull request by mribbons**

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

File: src/process/process.hh [hunk 1]
```diff
@@ -185,7 +185,7 @@ namespace SSC {
 
     int wait () {
       do {
-        std::this_thread::yield();
+        msleep(256);
```
Comment (chicoxyzzy) [inline src/process/process.hh:188]: maybe we can use something similar to this?

https://en.cppreference.com/w/cpp/thread/yield

Comment (chicoxyzzy) [reply src/process/process.hh:188]: There's https://en.cppreference.com/w/cpp/thread/sleep_for which looks like what we need

Comment (jwerle) [reply src/process/process.hh:188]: That is exactly what `msleep()` does :)

https://github.com/socketsupply/socket/blob/master/src/common.hh#L970-L972

Comment (chicoxyzzy) [reply src/process/process.hh:188]: oh great!

**Review by chicoxyzzy — APPROVED** [review]

**Review by chicoxyzzy — APPROVED** [review]

File: src/process/process.hh (hunk 1 above)
Comment (mribbons) [reply src/process/process.hh:188]: @chicoxyzzy The `yield()` + loop approach might be the right approach for `msleep()`, but it's worth noting that `yield()` alone wasn't working here in terms of lower CPU usage. `msleep()` does call `yield()`, but it also calls `sleep()`.

**Pull request merged**